ETL.Test/Repository/DBConnectionTest.cs

```diff
@@ -0,0 +1,98 @@
+using NUnit.Framework;
+using ETL.Repository;
+using System.Collections.Generic;
+
+namespace ETL.Repository.Test
+{
+    public class DBConnectionTest
+    {
+        private class TestClass
+        {
+            public string str;
+            public int num;
+
+            public TestClass(string str, int num)
+            {
+                this.str = str;
+                this.num = num;
+            }
+        }
+
+        [SetUp]
+        public void Setup()
+        {
+
+        }
```

Comment:
Does this framework require a setup or can it be removed?

Reply:
I wanted to get in my actual changes in first so we can start tying me and Alex's work together.  I'm switching this over to the Xunit framework once merged.  (Xunit doesn't have a Setup)